mix.exs

```diff
@@ -100,6 +101,8 @@ defmodule Livebook.MixProject do
       {:aws_signature, "~> 0.3.0"},
       {:ecto, "~> 3.9.0"},
       {:phoenix_ecto, "~> 4.4.0"},
+      {:mint_web_socket, "~> 1.0.0"},
+      {:livebook_proto, path: "proto"},
```

Comment:
@josevalim this won't work when publishing to Hex, no?

Reply:
Damn it, I forgot about this. I think we can remove the dependency from here and add it to elixirc_paths, right?

Reply:
Sounds good to me! Then we also need to add the .ex files to

https://github.com/livebook-dev/livebook/blob/b911bd71ed7dc7e59407af570b9c7d2e884acb66/mix.exs#L52

Reply:
Ah, but we also need to depend on `:protobuf`.

Reply:
Both updated